tests/cnf/ran/powermanagement/tests/cpufreq.go

```diff
@@ -24,11 +25,18 @@ var _ = Describe("CPU frequency tuning tests change the core frequencies of isol
 			desiredIsolatedCoreFreq = performancev2.CPUfrequency(2200002)
 			originalIsolatedCPUFreq performancev2.CPUfrequency
 			originalReservedCPUFreq performancev2.CPUfrequency
-			err                     error
 		)
 
 		BeforeEach(func() {
 
+			By("Checking that OCP version is 4.16 or later")
+			isGreaterOrEqual, err := rancluster.CompareOCPVersionWithCurrent(raninittools.Spoke1APIClient,
```

Comment:
My suggestion is to do it the same way we do in TALM tests to be consistent within out own dir for any version checks: https://github.com/openshift-kni/eco-gotests/blob/abbd21ba1908f1d814e692abdff419fdff2c9280/tests/cnf/ran/talm/tests/talm_backup.go#L26 

@klaskosk I would suggest to initialize some common variables at ran level that are applicable to all or most of the ran test suites. And I think spoke version is one of those. And move the version comparison talm helper to ran helper. 

I think this approach is better because spoke version is static throughout the test, and there will probably be more scenarios like this in other test suites. Plus this way, we keep our changes under RAN dir. :)

Reply:
@yliu127 I think the advantage of `CompareOCPVersionWithCurrent` is that it moves the version handling to another library that properly considers cases such as z-stream and `-rc` versions. Maybe we can add a different helper that does the actual version comparison so that we can use it separate from checking the OCP version? 

But yes I think it would be good to have a variable in `ranparam` for the OCP version that way we can compare throughout the test without checking again.

Reply:
@klaskosk our helper also handles zstreams, but yes using an existing library is nice. 

In this case, I would suggest do this in two steps: 
- ranparam change so Josh can use our helper 
- refactor our helper to use existing library

What do you think? The reason for refactoring our helper is because our desgin allows us to specify the start and the end releases in one call.